src/components/main.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="x-list x-list--vertical">
+    <Redirection />
+
+    <template v-if="!$x.redirections.length">
+      <div class="x-list x-list--padding-03">
+        <div
+          v-if="$x.totalResults === 0 || $x.spellcheckedQuery"
+          class="x-list x-list--padding-06 x-list--padding-bottom"
+        >
+          <SpellcheckMessage />
+          <NoResultsMessage />
+        </div>
+      </div>
+
+      <Results />
```

Comment:
Not for this PR, but something to take into account probably in the track pending query task. If you scroll fast tracking is skipped until you stop scrolling due to the debounce.

Reply:
noted there https://searchbroker.atlassian.net/browse/EX-4962